storage/gcp/bucket/variables.tf

```diff
@@ -0,0 +1,162 @@
+variable "bucket_name" {
+  description = "The name of the bucket."
+  type        = string
+}
+
+variable "alternative_location" {
+  description = "Alternative location for the bucket, by default it will use the project region."
+  type        = string
+  default     = null
+}
```

Comment:
```suggestion
variable "location" {
  description = "Location for the bucket, by default it will use the project region."
  type        = string
  default     = null
}
```